dvc/command/version.py

```diff
@@ -46,13 +45,11 @@ def run(self):
             # `dvc config cache.shared group`.
             if os.path.exists(repo.cache.local.cache_dir):
                 info.append(
-                    "Cache: {}".format(self.get_linktype_support_info(repo))
+                    "Cache types: " + self.get_linktype_support_info(repo)
                 )
                 if psutil:
                     fs_type = self.get_fs_type(repo.cache.local.cache_dir)
-                    info.append(
-                        f"Filesystem type (cache directory): {fs_type}"
-                    )
+                    info.append(f"Directory cache: {fs_type}")
```

Comment:
`Directory cache` feels a bit off here. Seems like it should be `Cache directory:` or maybe just `Cache:`

Reply:
+1 for `Cache:`

Reply:
Also, we could change the order of this to be just above `Workspace`.